public/app/types/dashboard.ts

```diff
@@ -53,6 +53,7 @@ export interface DashboardMeta {
   publicDashboardUid?: string;
   publicDashboardEnabled?: boolean;
   dashboardNotFound?: boolean;
+  isEmbedded?: boolean;
```

Comment:
I would name this "vscode" or "editorEmbedded"    just the "isEmbedded"  for this editing view mode is confusing.

Reply:
It's not VS code specific, any other editor can use it, I guess renaming to editorEmbed could work.